client/src/modules/Mentor/components/MentorDashboard/MentorDashboard.test.tsx

```diff
@@ -31,22 +30,20 @@ const PROPS_MOCK: CoursePageProps = {
     id: 400,
   } as Course,
   params: {},
+  mentorId: 1000,
+  studentsCount: 3,
 };
 
 describe('MentorDashboard', () => {
-  const useMentorStudentsCountMock = useMentorStudentsCount as jest.MockedFunction<typeof useMentorStudentsCount>;
-
   it('should render instructions when mentor has not students for this course', () => {
```

Comment:
Not related to your changes, but can we fix a typo here? `has not` => `has no`

Reply:
Fixed, thank you